frontend/packages/webterminal-plugin/integration-tests/OWNERS

```diff
@@ -0,0 +1,6 @@
+reviewers:
+  - hemantsaini-7
```

Comment:
Remove `hemantsaini-7` as he is not part of Openshift org anymore. You can add `sanketpathak` as a reviewer as well.

Reply:
Hi @vikram-raj ,

Updated the PR.